src/datepicker/datepicker.js

```diff
@@ -436,33 +521,27 @@ export default class Datepicker<T = Date> extends React.Component<
               }
               {...popoverProps}
             >
-              <InputWrapper {...inputWrapperProps}>
-                <InputComponent
-                  aria-disabled={this.props.disabled}
-                  aria-label={
-                    this.props['aria-label'] ||
-                    (this.props.range
-                      ? locale.datepicker.ariaLabelRange
-                      : locale.datepicker.ariaLabel)
-                  }
-                  error={this.props.error}
-                  positive={this.props.positive}
-                  aria-describedby={this.props['aria-describedby']}
-                  aria-labelledby={this.props['aria-labelledby']}
-                  aria-required={this.props.required || null}
-                  disabled={this.props.disabled}
-                  size={this.props.size}
-                  value={this.state.inputValue}
-                  onFocus={this.open}
-                  onBlur={this.handleInputBlur}
-                  onKeyDown={this.handleKeyDown}
-                  onChange={this.handleInputChange}
-                  placeholder={placeholder}
-                  mask={this.getMask()}
-                  required={this.props.required}
-                  clearable={this.props.clearable}
-                  {...inputProps}
-                />
+              {}
+              <InputWrapper
+                {...inputWrapperProps}
+                $separateRangeInputs={
+                  this.props.range && this.props.separateRangeInputs
+                }
+              >
+                {this.props.range && this.props.separateRangeInputs ? (
+                  <>
+                    <div style={{width: '100%', marginRight: '8px'}}>
```

Comment:
We may want to extract this div and below out into styled components. Do you think developers will need to customize them?